src/com/biffweb/impl/auth.clj

```diff
@@ -226,7 +226,7 @@
       {:status 303
        :headers {"location" app-path}
        :session (assoc session :uid (or existing-user-id
-                                        (get-user-id db email)))}
+                                        (get-user-id (xt/db db) email)))}
```

Comment:
The newly created user doesn't exist in this `db` so it's needed to find from the newest version.

Reply:
This throws an exception; it should be `(xt/db node)`. I just pushed the fix.

Reply:
Ah. Right. I didn't test this code but I wanted exactly what you said.